test/e2e/haproxy_timeouts_test.go

```diff
@@ -1,3 +1,4 @@
+//go:build e2e
```

Comment:
The 4.9 branch is built using Go 1.16, and `//go:build` is a Go 1.17 artifact.  The original PR didn't add this, and it isn't needed in 4.9, so better if the backport not pull in this gratuitious change.

Reply:
Good catch. I have my editor configured to automatically run gofmt when I save files, and it looks like it added this when I resolved the merge conflicts. This should be fixed now.